src/theme/components/Link.ts

```diff
@@ -25,6 +25,6 @@ export const Link = {
         color: 'brand.light.secondary',
         textDecorationColor: 'brand.light.secondary'
       }
-    }
+    },
```

Comment:
check that your prettier config is enabled, this comma should not be appearing as we have `"trailingComma": "none",` on the `.prettierrc` file, to avoid issues in the future